pages/linux/pacman-key.md

```diff
@@ -0,0 +1,36 @@
+# pacman-key
+
+> Wrapper script for GnuPG used to manage pacman’s keyring.
+> More information: <https://man.archlinux.org/man/pacman-key>.
+
+- Initialize the pacman keyring:
+
+`sudo pacman-key --init`
+
+- Add the default ArchLinux keys:
+
+`sudo pacman-key --populate {{archlinux}}`
+
+- List keys from the public keyring:
+
+`pacman-key --list-keys`
+
+- Add the specified keys:
+
+`sudo pacman-key --add {{path/to/keyfile}}`
+
+- Receive a key from a key server:
+
+`sudo pacman-key --recv-keys "{{key_id}}"`
+
+- Print the fingerprint of a specific key:
+
+`pacman-key --finger "{{key_id}}"`
+
+- Sign an imported key locally:
+
+`sudo pacman-key --lsign-key "{{key_id}}"`
```

Comment:
Sign with what?

Reply:
From the Arch Wiki <https://wiki.archlinux.org/title/Pacman/Package_signing>:
> you must locally sign the imported key:
> # pacman-key --lsign-key keyid
> You now trust this key to sign packages.

From <https://www.gnupg.org/gph/en/manual/r1020.html>:
> lsign-key — locally sign a public key with a private key

I think this is similar to trust an imported key. :thinking: Looks like a common practice.